### PR TITLE
Added helper methods to set up the ACLs and Authorizations for testing.

### PR DESCRIPTION
**Pull request by peichman-umd**
Note: building this branch still fails due to the `java.lang.VerifyError: Bad type on operand stack` error. But I wanted to share this with the team to get your input.

### CODE REVIEW

**Comment by awoods** [comment]
I am not seeing a `VerifyError`. Rather, when I run `mvn clean install` locally with this PR I get:
```bash
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.042 sec <<< FAILURE! - in org.fcrepo.integration.auth.webac.WebACRecipesIT
test(org.fcrepo.integration.auth.webac.WebACRecipesIT)  Time elapsed: 3.413 sec  <<< ERROR!
java.io.IOException: POST to http://localhost:33012/rest/ to create /acls/01/acl.ttl FAILED
        at org.fcrepo.integration.auth.webac.WebACRecipesIT.ingestTurtleResource(WebACRecipesIT.java:145)
        at org.fcrepo.integration.auth.webac.WebACRecipesIT.ingestAcl(WebACRecipesIT.java:96)
        at org.fcrepo.integration.auth.webac.WebACRecipesIT.setUp(WebACRecipesIT.java:73)
```

**Comment by peichman-umd** [comment]
@awoods, the VerifyError is still occurring for me when I run the test through Eclipse. The IOException causing the build failure is happening because the request is not returning a 201 response (its a 500 instead), see [line 144](https://github.com/fcrepo4-labs/fcrepo-module-auth-webac/pull/26/files#diff-dce4acc4ec190b8a7367f56783112ed4R144)

**Comment by awoods** [comment]
Running through Eclipse??!!
Give the command line a shot.

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 1]
```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
...
+
+        return aclURI;
+    }
+
+    /**
+     * Convenience method to POST the contents of a Turtle file to the repository to create a new resource. Returns
+     * the HTTP response from that request. Throws an IOException if the server responds with anything other than a
+     * 201 Created response code.
+     *
+     * @param credentials
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Remove these empty @param statements.

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 2]
```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
...
+        ingestAcl(credentials, "/acls/04/acl.ttl", "/acls/04/auth1.ttl", "/acls/04/auth2.ttl");
+        ingestAcl(credentials, "/acls/05/acl.ttl", "/acls/05/auth_open.ttl", "/acls/05/auth_restricted.ttl");
+
+        logger.debug("setup complete");
+    }
+
+    /**
+     * Convenience method to create an ACL with 0 or more authorization resources in the respository.
+     *
+     * @param credentials
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Remove these empty @param statements.

Comment (ajs6f) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: I'm inclined to think that there must be some way to have the IDE do this, but I have to admit that I don''t know quite how.

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 3]
```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
 
     @Before
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: If this "setUp" is intended to be run once before any @Tests, then it should be annotated with @BeforeClass... and therefore be static.

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 4]
```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
...
+     * @return URI of the created ACL
+     * @throws ClientProtocolException
+     * @throws IOException
+     */
+    private String ingestAcl(final String credentials, final String aclResourcePath,
+            final String... authorizationResourcePaths) throws ClientProtocolException, IOException {
+
+        // create the ACL
+        final HttpResponse aclResponse = ingestTurtleResource(credentials, aclResourcePath, serverAddress);
+        System.err.println(aclResponse.getStatusLine());
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: May as well use the `logger`.

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 5]
```diff
@@ -65,27 +67,84 @@ public WebACRecipesIT() {
...
+
+        // get the URI to the newly created resource
+        final String aclURI = aclResponse.getFirstHeader("Location").getValue();
+
+        // add all the authorizations
+        for (final String authorizationResourcePath : authorizationResourcePaths) {
+            final HttpResponse authzResponse = ingestTurtleResource(credentials, authorizationResourcePath, aclURI);
+            System.err.println(authzResponse.getStatusLine());
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: May as well use the `logger`.

**Comment by awoods** [comment]
Superseded by: https://github.com/fcrepo4-labs/fcrepo-module-auth-webac/pull/27

**Pull request closed without merging**